### PR TITLE
Propose to clarify

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/8317

### CODE REVIEW

File: Teams/Exchange-Teams-interact.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ The following table provides a helpful quick reference to feature availability b
 
 <sup>4</sup> Teams users with on-premises Exchange mailbox may use voicemail with Teams and receive voicemail messages in Outlook, but voicemail messages will not be available to view or play within the Teams client.
 
-<sup>5</sup> If one of the owners of a team can add connectors, everyone else in that team will be able to do so, even if their mailboxes are homed on-premises.
+<sup>5</sup> If one of the owners of a team can add connectors, everyone else in that team will be able to do so, no matter if their mailbox is homed on-premises or online.
```
Comment (JohanFreelancer9) [inline Teams/Exchange-Teams-interact.md:58]:
```suggestion
<sup>5</sup> If one of the owners of a team can add connectors, everyone else in that team will be able to do so, irrespective of whether their mailbox is homed on-premises or online.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - suggestion only.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by get-itips** [comment]
@dstrome

**Comment by get-itips** [comment]
@dstrome

**Comment by get-itips** [comment]
@tiburd Tina, can you help me with the review? Thank you

**Pull request merged**